app/views/users/show.html.erb

```diff
@@ -147,6 +147,14 @@
         <td><%= @user.metric 'E' %></td>
       </tr>
     </table>
+    
+    <% if current_user&.id == @user.id || current_user&.is_moderator %>
+    <table class="table is-full-width">
+      <tr>
+	<td>User since <%= @user.created_at %></td>
```

Comment:
I can imagine 2 potential improvements to the display format, but since this is already fully working I recommend merging as is, and raising these 2 improvements as separate issues if desired. If you like either of them I'm happy to raise them myself - just let me know.

1. Make the final space between the time and the timezone a [non-breaking space](https://en.wikipedia.org/wiki/Non-breaking_space), so when width is limited it displays as:
    ```text
    User since 2024-05-11
    12:51:22 UTC
    ```
    rather than:
    ```text
    User since 2024-05-11 12:51:22
    UTC
    ```
2. Show a less accurate but easier to read time elapsed, for consistency with other datetimes, with the full datetime shown on click or tap. This could use the `time_ago_in_words` function to show:
    ```text
    User for 16 days
    ```
    changing when clicked to:
    ```text
    User since 2024-05-11 12:51:22 UTC
    ```

Reply:
Both of these are definite improvements -- please do raise them, thanks!  (How do we do the first part -- grab the string, work back from the end to the first space, and then replace it and return the new string?  Or do we have manipulation functions for times/dates/timezones that we can use to extract the pieces and build the string we want?)

Reply:
I've now raised #1344 and #1345

As for how to approach the non-breaking space, your first suggestion is definitely possible (and is the only way I currently know how to do it). Ideally we would use your second suggestion, which would be more robust in case of other QPixel instances choosing to use different datetime formats that might be in a different order or might use different separators than spaces. I imagine Ruby has such built in functions (many languages do) but I'm not familiar with them yet.